docs/articles/test-workflows-reports.md

```diff
@@ -0,0 +1,64 @@
+# Test Workflows Reports: JUnit Processing and Visualization
+
+Testkube provides comprehensive support for processing and visualizing JUnit XML reports generated during Test Workflows.  
+This page explains how Testkube handles these reports and presents the information to users.
+
+## JUnit Report Extraction
+
+Testkube automatically scans all artifacts for `.xml` files that are valid [JUnit XML reports](https://github.com/testmoapp/junitxml).  
+It then parses their contents to make them available for various reporting and visualization features.
```

Comment:
add a "Read More" link to https://docs.testkube.io/articles/test-workflows-artifacts#junit-report-extraction ?